src/context/ThemeContext.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState, useEffect, useContext, createContext } from 'react';
+
+export const ThemeContext = createContext();
+
+export function ThemeProvider({ children }) {
+  const [theme, setTheme] = useState('light');
+
+  const toggleTheme = () => {
+    if (theme !== 'dark') {
+      localStorage.setItem('theme', 'dark');
+      setTheme('dark');
+    } else {
+      localStorage.setItem('theme', 'light');
+      setTheme('light');
+    }
+  };
+
+  useEffect(() => {
+    const localTheme = localStorage.getItem('theme');
```

Comment:
We could possibly move this logic inside of the useState on line 6

Reply:
Thank you! The block felt a bit clunky to me when writing, too, so this suggestion is especially helpful. Dropped the `useEffect` and refactored the redundancies in the `toggleTheme` in [`faa5f65` (#3)](https://github.com/jlaurentpdx/guestbook/pull/3/commits/faa5f65ec7532e62ceedabe14aad3c4e167c483a).